lib/ddtrace/contrib/active_support/notifications/subscriber.rb

```diff
@@ -36,7 +36,7 @@ def on_subscribe(&block)
             # Should be triggered by the inheriting class.
             def subscribe!
               return subscribed? if subscribed? || on_subscribe_block.nil?
-              on_subscribe_block.call
+              instance_eval(&on_subscribe_block)
```

Comment:
Please see my comment on the previous review for more details: I still do not agree with this change.

Reply:
Responded to it: https://github.com/DataDog/dd-trace-rb/pull/435#discussion_r193062309